assets/queries/cloudFormation/CustomSSLCertificates/query.rego

```diff
@@ -0,0 +1,15 @@
+package Cx
+
+CxPolicy [result] {
+  resource := input.document[i].Resources[name]
+  certificateSource := input.document[i].Resources[name].Properties.DistributionConfig.ViewerCertificate.certificateSource
+  expectedvalue := "cloudfront"
+  certificateSource == expectedvalue
```

Comment:
should use contains() instead of == operator

Reply:
Because the name shouldn't be like "%cloudfront%", which means the string might not be exactly 'cloudfront'

Reply:
Hi João :)
After a deep research I found that the other was not accurate. There are only 4 CertificateSource exact types: cloudfront, iam, acm and UNKNOWN_TO_SDK_VERSION (this one is not so used).

for cloudFront, if you ran the follow cmd to checkout the a well implemented distribution config you'll get the follow:
$ aws cloudfront get-distribution-config --id xxxxxxxxxx
{
    "DistributionConfig": {
        "ViewerCertificate": {
            "CertificateSource": "cloudfront",
            "CloudFrontDefaultCertificate": true,
            "MinimumProtocolVersion": "SSLv3"
        },
        
This is why I kept only cloudfront and not %cloudfront% :)